app/views/users/index.html.erb

```diff
@@ -1,8 +1,5 @@
-<h1>Lise of users with number of their posts</h1>
-<ul>
-  <% @users.each do |user| %>
-    <li>
-      <%= link_to user.name, user_path(user) %>
-    </li>
+<div class="container p-2">
+  <%@users.each do |user| %>
+    <%= render "shared/user_header", user: user%>
   <% end %>
-</ul>
```

Comment:
- [ ] Good job styling the user element on the page, however, you need to center the elements on the page to follow [the wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png). Kindly also make it larger to cover more width on the page.

  ![isos](https://user-images.githubusercontent.com/81468675/230176525-758b8b53-4a74-4ec6-94a3-685377b67900.jpg)